ninja-core/src/main/java/ninja/template/TemplateEngineFreemarker.java

```diff
@@ -333,7 +335,23 @@ public String getContentType() {
 
     @Override
     public String getSuffixOfTemplatingEngine() {
-        return FILE_SUFFIX;
+        return fileSuffix;
+    }
+    
+    /**
+     * Will override the default file suffix for freemarker templates.
+     * 
+     * @param fileSuffix file extension
+     */
+    public void setSuffixOfTemplatingEngine(String fileSuffix) {
```

Comment:
Would it be possible to init the fileSuffix via a variable in application.conf? That way we can set the fileSuffix in the constructor of this class. More importantly we can then set the FILE_SUFFIX to final - otherwise we might run into threading issues...